components/navbar.vue

```diff
@@ -0,0 +1,207 @@
+<template>
+  <nav>
+    <div class="box" @click="play" @keydown="play">
+      <div class="burgers b1"></div>
+      <div class="burgers b2"></div>
+      <div class="burgers b3"></div>
+    </div>
+    <div v-if="show" class="container">
+      <div class="circle"></div>
+      <div class="menu_container">
+        <ul class="menu">
+          <li class="list_item" @click="pause" @keydown="pause">
+            <nuxt-link to="/">Home</nuxt-link>
+          </li>
+          <li class="list_item" @click="pause" @keydown="pause">
+            <nuxt-link to="/projects">Projects</nuxt-link>
+          </li>
+          <li class="list_item" @click="pause" @keydown="pause">
+            <nuxt-link to="/">Blogs</nuxt-link>
+          </li>
+          <li class="list_item" @click="pause" @keydown="pause">
+            <nuxt-link to="/">About Us</nuxt-link>
+          </li>
+        </ul>
+        <div class="close" @click="pause" @keydown="pause">
+          <div class="cross cross1"></div>
+          <div class="cross cross2"></div>
+        </div>
+      </div>
+    </div>
+  </nav>
+</template>
+
+<script>
+import { defineComponent } from '@nuxtjs/composition-api';
+import gsap from 'gsap';
```

Comment:
for gsap use it as a plugin and then import it from there not directly as an npm package. We can use [npm package](https://www.npmjs.com/package/nuxt-gsap-module) for making the task easier take a look around if it seems alright i just found it